### DISCUSSION TITLE
各位老板 谷歌翻译可以用 翻译文档不能用是什么情况

### DISCUSSION THREAD

**Discussion by ricekil** [YiiGuxing/TranslationPlugin#2201] [Q&A]
梯子好使，hosts已映射 
![image](https://user-images.githubusercontent.com/41065897/195279564-3f4e02ca-ab65-46b9-93d5-b4f34d98052d.png)
![image](https://user-images.githubusercontent.com/41065897/195279633-21ed43f3-9752-4521-a436-37ef1aca8f69.png)
![image](https://user-images.githubusercontent.com/41065897/195279686-bd3ca976-5ad8-4a9d-9e85-cac91d36f3cb.png)

**Comment by YiiGuxing** [comment]
无法解析？应该是插件版本的问题，更新一下插件。
